Antidote/CallsManager.m

```diff
@@ -437,6 +441,9 @@ - (void)switchViewControllerForCall:(OCTCall *)call
 {
     AALogVerbose(@"%@", call);
 
+    UIDevice *currentDevice = [UIDevice currentDevice];
+    currentDevice.proximityMonitoringEnabled = (call.status == OCTCallStatusActive);
```

Comment:
It should be enabled for dialing screen as well. Also if would be nice to enable proximityMonitoring only when loud speaker is off and user uses ear speaker.